OpenOversight/app/templates/profile.html

```diff
@@ -0,0 +1,98 @@
+{% extends "base.html" %}
+{% block content %}
+
+<div class="container" role="main">
+
+
+  <div class="page-header">
+    <h1>Profile: {{ user.username }}</h1>
+  </div>
+
+  <div class="row">
+    <div class="col-sm-6 col-md-4">
+      <div class="thumbnail">
+        <div class="caption">
+          <h3>User Statistics</h3>
+          <table class="table table-hover">
+            <tbody>
+              <tr>
+                <td><b>Number of images classified</b></td>
+                <td>{{ user.classifications|length }}</td>
+              </tr>
+              <tr>
+                <td><b>Number of officers found</b></td>
+                <td>{{ user.tags|length }}</td>
+              </tr>
+            </tbody>
+          </table>
+
+          <p><a href="{{ url_for('main.leaderboard') }}"
+                class="btn btn-primary" role="button">Show leaderboard</a></p>
+
+          {% if current_user.is_administrator and user.is_disabled == False %}
```

Comment:
@r4v5: That's a good point - modified in commit c054f0959379f17ea60a57f6d419c3664e59a4e7

@freddymartinez9: Also added an if statement in c054f0959379f17ea60a57f6d419c3664e59a4e7 such that the toggle button is not displayed if the user is an admin (it wouldn't _really_ do anything anyway - their classifications/tags won't get added if their account is disabled but they are administrators so they'd be able to toggle their own account back to enabled).